test/runtests.jl

```diff
@@ -1,33 +1,51 @@
 
 using Test
+using HTTP, Gumbo, AbstractTrees, URIs
```

Comment:
I would recommend splitting this into dedicated files that reflect the corresponding `src/*.jl` file!

You should not directly include code --> use named imports to get the functions you need INSIDE of the relevant files, eg, in `test/utils.jl` you'll have imports at the top like `using DocsScraper: find_duplicates` (you can important many things by comma separating them)

Reply:
Normal package imports like the below can stay in the main `test/runtests.jl`

You then call all the test files like this:
```julia
@testset "DocParser.jl begin
include("crawl.jl")
... # other files
end
```